CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+<!-- @license CC0-1.0 -->
```

Comment:
Een waardevolle toevoeging voor dit document: uitleg welke van de 4 licenties die we in deze PR gebruiken (CC0-1.0, CC-BY-4.0, EUPL-1.2 met of zonder Robbert Broersma; plus waarom hij wel een jaartal en wij niet) in welk soort gevallen toegepast moet worden op een nieuw bestand.

Reply:
Mooi beschreven, beantwoordde in elk geval al mijn vragen!